core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -1160,13 +1160,16 @@ private[spark] object SparkSubmitUtils {
     val br: IBiblioResolver = new IBiblioResolver
     br.setM2compatible(true)
     br.setUsepoms(true)
+    val defaultInternalRepo : Option[String] = sys.env.get("DEFAULT_ARTIFACT_REPOSITORY")
+    br.setRoot(defaultInternalRepo.getOrElse("https://repo1.maven.org/maven2/"))
```

Comment:
Is the default value always `https://repo1.maven.org/maven2/` for all user environments?

Reply:
Yes. That is the default for `IBiblioResolver`.